evm_arithmetization/src/generation/linked_list.rs

```diff
@@ -8,15 +9,35 @@ use crate::util::u256_to_usize;
 use crate::witness::errors::ProgramError;
 use crate::witness::errors::ProverInputError::InvalidInput;
 
+pub const ACCOUNTS_LINKED_LIST_NODE_SIZE: usize = 4;
+pub const STORAGE_LINKED_LIST_NODE_SIZE: usize = 5;
+
+pub(crate) trait LinkedListType {}
+#[derive(Clone)]
+/// A linked list that starts from the first node after the special node and iterates for ever.
```

Comment:
```suggestion
/// A linked list that starts from the first node after the special node and iterates forever.
```

Reply:
also we may want more clarity on what is the special node here (and below for bounded version)